AutoCollection/PreAggregatedMetrics.ts

```diff
@@ -172,8 +170,8 @@ class AutoCollectPreAggregatedMetrics {
             currentCounter.time = +new Date;
             var intervalRequests = (currentCounter.totalCount - currentCounter.lastTotalCount) || 0;
             var elapsedMs = currentCounter.time - currentCounter.lastTime;
```

Comment:
This method and countDependency/countRequet methods have 95% similar code and perhaps could be refactored to remove duplication, in future.